app-backend/batch/src/main/scala/com/azavea/rf/batch/util/conf/Config.scala

```diff
@@ -74,24 +75,27 @@ trait Config {
     bucketName: String,
     targetProj: String
   ) {
-    def organizationUUID = UUID.fromString(organization)
-    def datasourceUUID = UUID.fromString(datasourceId)
-    def targetProjCRS = CRS.fromName(targetProj)
+    def organizationUUID: UUID = UUID.fromString(organization)
+
+    def datasourceUUID: UUID = UUID.fromString(datasourceId)
+
+    def targetProjCRS: CRS = CRS.fromName(targetProj)
+
     def bandByName(key: String): Option[Band.Create] =
       bandLookup.getClass.getDeclaredFields.toList.filter(_.getName == key).map { field =>
         field.setAccessible(true)
         field.get(bandLookup).cast[Band.Create] // safe shapeless cast, instead asInstanceOf call
       }.headOption.flatten
   }
 
-  case class Dropbox(appKey: String, appSecret: String) {
+  final case class Dropbox(appKey: String, appSecret: String) {
     lazy val appInfo = new DbxAppInfo(appKey, appSecret)
-    lazy val config  = new DbxRequestConfig("azavea/rf-dropbox-test")
+    lazy val config = new DbxRequestConfig("azavea/rf-dropbox-test")
 
     def client(accessToken: String) = new DbxClientV2(config, accessToken)
   }
 
-  case class Auth0(
+  final case class Auth0(
```

Comment:
I should note that this is not emitted as part of the Scapegoat analysis.

Reply:
Yeah, I'm not sure what to do here -- https://stackoverflow.com/a/16466541 gives a pretty good explanation of what's happening.

I think we're safe to ignore these warnings from the compiler -- we're not pattern matching on the config classes anywhwere, but I think it would be good to suppress them somehow then too.

The other alternative is if we have scapegoat ignore these ones (I don't think they're at much risk for being extended anyways).

Reply:
I dropped the `final` and added the `@SuppressWarnings` annotation.